common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1229,6 +1229,12 @@
     ],
     "sqlState" : "42710"
   },
+  "DUPLICATED_ARTIFACT" : {
+    "message" : [
+      "Duplicate Artifact: <normalizedRemoteRelativePath>. Artifacts cannot be overwritten."
```

Comment:
```suggestion
      "The artifact <normalizedRemoteRelativePath> already exists. Please choose a different name for the new artifact because it cannot be overwritten."
```

Reply:
Applied the suggestions. Thanks!